_targets.R

```diff
@@ -5,27 +5,63 @@ source("3_visualize/src/plot_timeseries.R")
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
-
+#site numbers
+sites = c("01427207", "01432160","01435000", "01436690", "01466500")
+#parameter codes for query
+parameterCd = '00010' 
+startDate="2014-05-01" 
+endDate="2015-05-01"
+#parameters for plotting
 p_width <- 12
 p_height <- 7
 p_units <- "in"
 
 p1_targets_list <- list(
   tar_target(
-    site_data,
-    download_nwis_data(),
+    create_directories,
+    create_dirs(sites),
+    format = "file"
+  ),
+  tar_target(
+    site_data_1,
+    download_nwis_site_data(sites[1], create_directories,parameterCd, startDate, endDate),
+    format = "file"
+  ),
+  tar_target(
+    site_data_2,
+    download_nwis_site_data(sites[2], create_directories,parameterCd, startDate=, endDate=),
+    format = "file"
+  ),
+  tar_target(
+    site_data_3,
+    download_nwis_site_data(sites[3], create_directories,parameterCd, startDate=, endDate=),
+    format = "file"
+  ),
+  tar_target(
+    site_data_4,
+    download_nwis_site_data(sites[4], create_directories,parameterCd, startDate=, endDate=),
+    format = "file"
+  ),
+  tar_target(
+    site_data_5,
+    download_nwis_site_data(sites[5], create_directories,parameterCd, startDate=, endDate=),
+    format = "file"
+  ),
+  tar_target(
+    site_data_concat,
+    concat_files_to_df()
   ),
```

Comment:
Here you have another solution to trying to have a more dynamic pipeline that would respond to varying folder contents which could be nice! But for this static pipelines example you want this target to _depend_ on the individual site downloads. So here you would want to revise your combine function to take in the individual download targets and return a combined data target. 

As an aside the targets package has a function to use a directory as an input to target which is kind of what you coded up here. So in the future keep in mind a more advanced feature is to rely on a directory as a target dependency - which is a great option in the case where there is a folder that people might manually put files in, etc.